tests/unit/date/core.js

```diff
@@ -144,17 +144,19 @@ test( "Days", 1, function() {
 	ok( date.days(), "Date days() returns" );
 } );
 
-test( "Months", 5, function() {
-	var date = $.ui.date( null, attributes ),
-		firstMonth = date.months( 1 )[ 0 ],
-		lastMonth = date.months( 1 )[ 1 ];
-
-	ok( firstMonth.first );
-	ok( !lastMonth.first );
-	ok( lastMonth.last );
-	ok( !lastMonth.first );
-
-	equal( firstMonth.month(), lastMonth.month() - 1 );
+test( "Months", 4, function() {
+	var date = $.ui.date( new Date( 2015, 11 - 1, 15 ), attributes ),
+		currentMonth = date.months( 1 )[ 0 ],
+		nextMonth = date.months( 1 )[ 1 ];
+
+	ok( currentMonth.first, "Current month marked as first" );
+	ok( !nextMonth.first, "Next month not marked as first" );
+	ok( nextMonth.last, "Next month marked as last" );
+
+	equal(
```

Comment:
Now that the values no longer depend on the current date, replace this with two "static" assertions:

```js
equal( currentMonth.month(), 11 );
equal( nextMonth.month(), 12 );
```
(or whatever the expected values are)